provider/openrouter/openrouter.go

```diff
@@ -51,14 +54,37 @@ func (p *Provider) ID() (id string) {
 	return "openrouter"
 }
 
+// ModelsList holds a list of models.
+type ModelsList struct {
+	Models []Model `json:"data"`
+}
+
+// Model holds a model.
+type Model struct {
+	// ID holds the model id.
+	ID string `json:"id"`
+	// Pricing holds the pricing information of a model.
+	Pricing Pricing `json:"pricing"`
+}
+
+// Pricing holds the pricing information of a model.
+type Pricing struct {
+	// Prompt holds the price for a prompt.
+	Prompt string `json:"prompt"`
+	// Completion holds the price for a completion.
+	Completion string `json:"completion"`
+	// Request holds the price for a request.
+	Request string `json:"request"`
+	// Image holds the price for a image.
+	Image string `json:"image"`
```

Comment:
Please be more detailed. I.e. I think `Prompt` should be the price for 1 million input tokens and `Completion` for 1 million output tokens, right? Please just check out a model in the web UI and then look at the same model in the JSON and see which numbers match up.

Reply:
On the web UI the costs for e.g. prompt is presented in $/1M tokens, but in the JSON response the value is $/token. Example: `01-ai/yi-large` 3$ per 1M tokens in the web UI vs. 0.000003 in the JSON

Reply:
ohhh cool. well then let's use that one then